packages/validation/src/user.ts

```diff
@@ -0,0 +1,3 @@
+export const USER_SOCIAL_WEBSITE = /^http(s?):\/\/.+/i;
+export const USER_SOCIAL_RESEARCHER_ID = /^[a-z]{1,3}-\d{4}-20[0-2]\d$/i;
```

Comment:
```suggestion
export const USER_SOCIAL_RESEARCHER_ID = /^[a-z]{1,3}-\d{4}-\d{4,}$/i;
```
yeah idk if this was specified in the ticket but if it was we should push back :D I don't want a regex that requires maintenance in a maximum of 8.5yrs :D